Makefile

```diff
@@ -19,7 +19,7 @@ PAK_CACHE=/tmp/.pak-cache
 SECURITY_USER_NAME := $(or $(SECURITY_USER_NAME), aws-broker)
 SECURITY_USER_PASSWORD := $(or $(SECURITY_USER_PASSWORD), aws-broker-pw)
 GSB_COMPATIBILITY_ENABLE_BETA_SERVICES :=true
-export GSB_SERVICE_CSB_GOOGLE_POSTGRES_PLANS = [{"name":"small","id":"85b27a04-8695-11ea-818a-274131861b81","description":"PostgreSQL with default version, shared CPU, minimum 0.6GB ram, 10GB storage","display_name":"small","cores":0.6,"storage_gb":10},{"name":"medium","id":"b41ee300-8695-11ea-87df-cfcb8aecf3bc","description":"PostgreSQL with default version, shared CPU, minimum 1.7GB ram, 20GB storage","display_name":"medium","cores":1.7,"storage_gb":20},{"name":"large","id":"2a57527e-b025-11ea-b643-bf3bcf6d055a","description":"PostgreSQL with default version, minimum 8 cores, minimum 8GB ram, 50GB storage","display_name":"large","cores":8,"storage_gb":50}]
```

Comment:
All the documentation I have found about SQL tiers, mark the `db-n1-standard-8` tier with 30GB of RAM, which I understand is the maximum value. Perhaps it would be worth adding it to the description even though it could give us problems in future updates if we do not pay attention to possible changes in Google Cloud policies. Or maybe I didn't find the proper documentation. Apologies beforehand if that happened.

Reply:
This is a good catch, you're right the minimum RAM is 30GB, I have updated this description. 
For context we add these plans for our own testing, and they are not intended for production use.
This may actually be an example of Google having increased the minimum ram for the service tier, and the description becoming out of date. 
We're moving towards the pattern of letting users create their own plans, rather than having pre-configured curated plans, so we shouldn't have the maintenance burden of updating plan descriptions - apart from this test example.